eng/pipelines/templates/steps/clean-maven-local-cache.yml

```diff
@@ -7,3 +7,9 @@ steps:
       arguments: >
         -MavenCacheFolder $(MAVEN_CACHE_FOLDER)
     continueOnError: true
+
+  - pwsh: |
```

Comment:
Should we just change the name of this template to be a bit clearer and move all the above steps you added into it too?

Reply:
Yeah, I think we can call this `post-job-maven-clean`, or something like that